functions/java-jre.bash

```diff
@@ -231,3 +261,47 @@ java_zulu_install_crypto_extension(){
   rm -rf "${policyTempLocation}"
   return 0
 }
+
+## Install AdoptOpenJDK using APT repository.
+##
+adoptopenjdk_install_apt(){
+  local adoptKey="/tmp/adoptopenjdk.asc"
+  if ! dpkg -s 'adoptopenjdk-8-hotspot-jre' >/dev/null 2>&1; then # Check if already is installed
+    echo -n "$(timestamp) [openHABian] Adding AdoptOpenJDK keys to apt... "
+    cond_redirect wget -qO "$adoptKey" https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public
+    if cond_redirect apt-key add "$adoptKey"; then
+      echo "OK"
+    else
+      echo "FAILED (keyserver)"
+      rm -f "$adoptKey"
+      exit 1;
+    fi
+    echo -n "$(timestamp) [openHABian] Installing AdoptOpenJDK... "
+    cond_redirect systemctl stop openhab2.service
+    cond_redirect update-alternatives --remove-all java
+    echo "deb http://adoptopenjdk.jfrog.io/adoptopenjdk/deb buster main" > /etc/apt/sources.list.d/adoptopenjdk.list
+    cond_redirect apt-get update
+    if cond_redirect apt-get --yes install adoptopenjdk-8-hotspot-jre; then echo "OK"; else echo "FAILED"; exit 1; fi
```

Comment:
does that take care of update-alternatives ?

Reply:
Yes

Reply:
I'm testing the different options personally to make sure that they work.

Just kidding on reinstall it doesn't I'm fixing it now

Reply:
It does now :)